### DISCUSSION TITLE
Welcome

### DISCUSSION THREAD

**Discussion by torhaa1** [ParanoidUser/yolo#171] [Subscribe]
Hi

**Comment by torhaa1** [comment]


**Reply by ParanoidUser** [reply]
Ahoy @t-haakens, welcome aboard!

The invitation to join the collaborator group has been sent to you. Thanks for your interest in this project!
Feel free to post your own questions to the community, or share your path to earning other badges. 😊 Let me know if you have any questions.

Regarding the Pair Extraordinaire badge, please take a look at the topic [Badge: Pair Extraordinaire](https://github.com/ParanoidUser/yolo/discussions/26). It is a step-by-step guide on how to earn it.

Cheers!

**Comment by ParanoidUser** [comment]
@t-haakens, sorry to see you leave the Collaborators. 😢 I hope you found it useful.
